include/godot_cpp/core/class_db.hpp

```diff
@@ -202,9 +229,9 @@ void ClassDB::_register_class(bool p_virtual, bool p_exposed) {
 		T::to_string_bind, // GDExtensionClassToString to_string_func;
 		nullptr, // GDExtensionClassReference reference_func;
 		nullptr, // GDExtensionClassUnreference unreference_func;
-		T::create, // GDExtensionClassCreateInstance create_instance_func; /* this one is mandatory */
+		create<T>, // GDExtensionClassCreateInstance create_instance_func; /* this one is mandatory */
```

Comment:
While it isn't necessary, I find this code easier to read when the function is preceded by `&`, which is also the style we mostly (with some exceptions) use in Godot. So, with the name change from above, this would be:

```suggestion
		&_create_instance_func<T>, // GDExtensionClassCreateInstance create_instance_func; /* this one is mandatory */
```